src/main/java/mods/eln/Eln.java

```diff
@@ -4793,7 +4793,8 @@ private void registerElectricalTool(int id) {
 
             ElectricalLampItem desc = new ElectricalLampItem(
                 name,
-                10, 8, 20, 15, 5, 50,// int light,int range,
+				//10, 8, 20, 15, 5, 50, old
+                10, 8, 20, 15, 5, 50,// int light,int range
```

Comment:
I'm not exactly sure why this was marked as different. I suspect that you changed something and then changed it back, but that GitHub noticed the old comment line and thought there were more changes than that.

Either that, or I'm blind to constants.

Reply:
I changed the code, it was causing bugs so I reverted it but never got rid of the //old comments, so it thinks they're unique coding changes when practically nothing was changed.